lib/node_modules/@stdlib/blas/base/dnrm2/test/test.dnrm2.js

```diff
@@ -53,6 +84,31 @@ tape( 'the function calculates the L2-norm of a vector', function test( t ) {
 	z = dnrm2( x.length, x, 1 );
 	t.strictEqual( z, 4.0, 'returns expected value' );
 
+	x = new Float64Array([ 1.0e150, 1.0e150, 1.0e150, 1.0e150 ]);
```

Comment:
```suggestion
	x = new Float64Array( [ 1.0e150, 1.0e150, 1.0e150, 1.0e150 ] );
```

@ShabiShett07 You are still missing spacing inside parentheses here and elsewhere throughout this PR.